### ISSUE TITLE
Replace micromatch github dependency

### ISSUE THREAD

**Issue by ghost** [micromatch/glob-fs#23]
Could you please replace the micromatch github- by a npm-dependency?
This leads to trouble by firewall restrictions or deployments through inhouse npm caches. Thx!

**Comment by jonschlinkert** [comment]
want to do a pr?

**Comment by henninga** [comment]
I could do a pr but all of the tests fails on my machine right after npm install

**Comment by marlonmarcello** [comment]
@jonschlinkert I have the same issue as @henninga 
This breaks CI/CD on Amplify and others.

**Comment by jonschlinkert** [comment]
yes, sure. want to do a PR? sorry for the lack of activity, I'll be spending a lot more time on GitHub going forward. 

thanks!

**Comment by marlonmarcello** [comment]
All good @jonschlinkert ! Appreciate everything that you've done.
I can't do a PR for the same reason that @henninga can't. I get a bunch of test fails.

**Comment by jonschlinkert** [comment]
> I can't do a PR for the same reason that @henninga can't. I get a bunch of test fails.

Well... lol someone will need to get those tests passing. I'd appreciate a PR from someone, but I can do it when I get a chance.

**Comment by rbitting** [comment]
Any update on this? I'm encountering the same issue. FWIW it doesn't seem like the unit tests pass with the current micromatch github package either 😅

Edit: Workaround until this is fixed is to just add a resolution for micromatch in the package.json. Example:

```
"resolutions": {
    "micromatch": "^4.0.2"
}
```
